tasks/client.yml

```diff
@@ -62,46 +62,56 @@
   notify: Restart BeeGFS client service
 
 - name: Make of copy of BeeGFS client config file if it doesn't exist
+  vars:
+    mgmt_host: "{{ item.mgmt_host | default(beegfs_mgmt_host) }}"
```

Comment:
Would it work if you iterated over the include of a file of tasks for mounting one client?  That might save a lot of replication of defining mgmt_host and repeating with_items.